app/controllers/games_controller.rb

```diff
@@ -1,4 +1,21 @@
 class GamesController < ApplicationController
-  # Add your GamesController code here
+  def create
+    game = Game.create(state: params[:state])
+    render json: game
+  end
 
+  def show
+    game = Game.find(params[:id])
+    render json: game
+  end
+
+  def update
+    game = Game.find(params[:id])
+    game.update(state: params[:state])
+    render json: game
+  end
+
+  def index
+    render json: Game.all
+  end
 end
```

Comment:
You should always ensure that you are whitelisting your parameters to only allow approved params to be used in the controller.

```ruby
class GamesController < ApplicationController
  #your game routes

  private
    def game_params
      params.permit(state: [])
    end
end
```